Assets/Tests/InputSystem/Plugins/XRTests.cs

```diff
@@ -728,6 +728,37 @@ public void Components_TrackedPoseDriver_RetainsPoseWhenTrackedDeviceRemoved()
         }
     }
 
+    [Category("Components")]
```

Comment:
```suggestion
    [Test]
    [Category("Components")]
```
This test won't actually execute without `[Test]`.